website/redirects.js

```diff
@@ -0,0 +1,1234 @@
+    permanent: true,
+  },
+  {
+    source: '/guides/advanced-scheduling/',
+    destination:
+      'https://learn.hashicorp.com/collections/nomad/advanced-scheduling',
+    permanent: true,
+  },
+  {
+    source: '/guides/external/(index.html)?',
+    destination: 'https://learn.hashicorp.com/collections/nomad/plugins',
+    permanent: true,
+  },
+  {
+    source: '/guides/external/lxc(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/plugin-lxc',
+    permanent: true,
+  },
+  {
+    source: '/guides/operations/upgrade/(index.html)?',
+    destination: '/docs/upgrade',
+    permanent: true,
+  },
+  {
+    source: '/guides/operations/upgrade/upgrade-specific(.html)?',
+    destination: '/docs/upgrade/upgrade-specific',
+    permanent: true,
+  },
+  {
+    source: '/guides/upgrade/(index.html)?',
+    destination: '/docs/upgrade',
+    permanent: true,
+  },
+  {
+    source: '/guides/upgrade/upgrade-specific(.html)?',
+    destination: '/docs/upgrade/upgrade-specific',
+    permanent: true,
+  },
+
+  {
+    source: '/guides/operations/deployment-guide(.html)?',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/production-deployment-guide-vm-with-consul',
+    permanent: true,
+  },
+  {
+    source: '/guides/operations/reference-architecture(.html)?',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/production-reference-architecture-vm-with-consul',
+    permanent: true,
+  },
+  {
+    source: '/docs/install/production/deployment-guide',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/production-deployment-guide-vm-with-consul',
+    permanent: true,
+  },
+  {
+    source: '/docs/install/production/reference-architecture',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/production-reference-architecture-vm-with-consul',
+    permanent: true,
+  },
+
+  // Enterprise
+
+  // Reorganized Enterprise into single pager
+  {
+    source: '/docs/enterprise/namespaces/(index.html)?',
+    destination: '/docs/enterprise#namespaces',
+    permanent: true,
+  },
+  {
+    source: '/docs/enterprise/quotas/(index.html)?',
+    destination: '/docs/enterprise#resource-quotas',
+    permanent: true,
+  },
+  {
+    source: '/docs/enterprise/preemption/(index.html)?',
+    destination: '/docs/enterprise#preemption',
+    permanent: true,
+  },
+  {
+    source: '/docs/enterprise/sentinel/(index.html)?',
+    destination: '/docs/enterprise#sentinel-policies',
+    permanent: true,
+  },
+  {
+    source: '/docs/enterprise/autopilot/(index.html)?',
+    destination: '/docs/enterprise#nomad-enterprise-platform',
+    permanent: true,
+  },
+
+  // Guide Catch-all Redirects
+  {
+    source: '/guides',
+    destination: 'https://learn.hashicorp.com/nomad',
+    permanent: true,
+  },
+  {
+    source: '/guides/:splat*',
+    destination: 'https://learn.hashicorp.com/nomad',
+    permanent: true,
+  },
```

Comment:
Just curious if this pattern can be expressed by one redirect like `/guides/(:splat*)?`